src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -47,6 +47,10 @@ public abstract class Sensor : IDisposable
         private IntPtr _sensorHandle = IntPtr.Zero;
         private IntPtr _listenerHandle = IntPtr.Zero;
 
+        /// <summary>
+        /// Read a sensor data synchronously.
+        /// </summary>
+        protected virtual void ReadData() { }
```

Comment:
`abstract` is more correct to enforce implementation it on base class
```suggestion
        protected abstract void ReadData() ;
```

Reply:
yes, true that.